vars/postgresql_12.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Best to keep an empty file here, so that it's less confusing.  The current version should ALWAYS be empty, as the defaults should be all set in `defaults/main.yml`.

Reply:
Maybe worth adding a comment in the file itself to clarify this, since one could probably wonder why the file is there at all.